modules/bullet/space_bullet.cpp

```diff
@@ -1318,7 +1318,10 @@ bool SpaceBullet::RFP_convex_world_test(const btConvexShape *p_shapeA, const btC
 		dispatcher->freeCollisionAlgorithm(algorithm);
 
 		if (contactPointResult.hasHit()) {
-			r_delta_recover_movement += contactPointResult.m_pointNormalWorld * (contactPointResult.m_penetration_distance * -1 * p_recover_movement_scale);
+			if(!(motion == btVector3(0,0,0) || motion ==NULL))
+			{
```

Comment:
There are some style issues here: the brace should be on the same line, and there should be a space after `if`. I recommend you [install clang-format](https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#using-clang-format-locally) to fix formatting automatically.

Once you've fixed issues in this pull request, you can amend your commit with fixes and force-push it to your fork. See [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html) for details.